README.md

```diff
@@ -1,17 +1,45 @@
 
 ![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
 
-# SuperTokens Website SDK
+# SuperTokens Javascript Frontend SDK
 
 <a href="https://supertokens.io/discord">
 <img src="https://img.shields.io/discord/603466164219281420.svg?logo=discord"
     alt="chat on Discord"></a>
-    
-## About
-This is a website / JavaScript SDK that is responsible for maintaining a SuperTokens session for website apps (SPA and non-SPA).
+
+SuperTokens adds **secure login and session management** to your apps. This is the Javascript Frontend SDK for SuperTokens. [More SDKs are available](https://github.com/supertokens) for frontend and backend e.g. Node.js, Go, Python, React.js, React Native, Vanilla JS, etc.
```

Comment:
Shuold mention that this only helps with session management?

Reply:
What do users need to do to implement login for their vanilla js website? (when they already have supertokens backend)

Reply:
We are working on that SDK (https://github.com/supertokens/supertokens-web-js) - it's not released yet.

But right now, they would use supertokens-website for sessions and call the APIs using `axios` or `fetch` for login.